### PR TITLE
Document working-group checkins and simplify the dreaded table.

### PR DESCRIPTION
**Pull request by davidtwco**

### CODE REVIEW

**Comment by Centril** [comment]
The new table looks fab! 💯

File: about/triage-meeting.md [hunk 1]
```diff
@@ -1,14 +1,37 @@
...
-YYYY-MM-DD" (with the proper date, of course).
+The triage meeting takes place on [Zulip](chat-platform.md). Look for a topic like
+"#54818 weekly meeting YYYY-MM-DD" or "weekly meeting YYYY-MM-DD" (with the proper date, of course).
 
+## Working group check-in
+This section contains the scheduled check-ins for working groups:
 
+- **2019-03-14:** [wg-rls-2.0], [wg-meta]
+- **2019-03-21:** [wg-self-profile], [wg-rfc-2229]
+- **2019-03-28:** [wg-self-profile], [wg-rfc-2229]
```
Comment (nikomatsakis) [inline about/triage-meeting.md]: Nit: I expect this to be sorted in "most recent to last" -- I'd probably just make one big list?

I also think that if we should adopt a convention in the working-groups directory like

```
working-group/wg-foo/updates/2019.03.14.md
```

which would contain the update? Maybe? (Or maybe it's too much)

The idea is though that (a) if you are browsing a wg you can quickly skim the last few updates and (b) we can link to them from there. 

Anyway doesn't have to be part of this PR.

File: about/triage-meeting.md [hunk 2]
```diff
@@ -12,9 +12,9 @@ The triage meeting takes place on [Zulip](chat-platform.md). Look for a topic li
 ## Working group check-in
 This section contains the scheduled check-ins for working groups:
 
-- **2019-03-14:** [wg-async-await], [wg-llvm]
 - **2019-03-21:** [wg-nll], [wg-traits]
 - **2019-03-28:** [wg-parallel-rustc], [wg-pgo]
+- **2019-04-24:** [wg-rls-2.0], [wg-meta]
```
Comment (nikomatsakis) [inline about/triage-meeting.md]: Maybe we do want to unify these into one list, though, so that no update is needed?

Comment (davidtwco) [reply about/triage-meeting.md]: I'm not sure I understand what you mean. 

I spent a little bit of time trying to think of a nice way to encode the list (or refer to the row number of the big table in the README) and then work out what it should be for this coming meeting, but I couldn't think of anything that was sufficiently easy.

File: about/triage-meeting.md (hunk 1 above)
Comment (davidtwco) [reply about/triage-meeting.md]: I've updated the ordering.

File: about/triage-meeting.md (hunk 2 above)
Comment (nikomatsakis) [reply about/triage-meeting.md]: Oh, I just meant having a list by date, and assuming people could tell which dates were in the future, and which were in the past.

**Comment by nikomatsakis** [comment]
anyway let's get this merged! r=me once rebased (cc @davidtwco)

**Comment by davidtwco** [comment]
Oops! Didn't see this needed a rebase.

**Pull request merged**